library/spdm_requester_lib/libspdm_req_vendor_request.c

```diff
@@ -57,7 +57,7 @@ libspdm_return_t libspdm_try_vendor_send_request_receive_response(
         (*resp_size != 0 && resp_data == NULL)
         ) {
         status = LIBSPDM_STATUS_INVALID_PARAMETER;
-        goto done;
+        return status;
```

Comment:
Just `return LIBSPDM_STATUS_INVALID_PARAMETER;`

Reply:
Thanks. I have changed the code.